hpedockerplugin/hpe_storage_api.py

```diff
@@ -159,6 +159,26 @@ def volumedriver_create(self, name, opts=None):
                     LOG.error(msg)
                     return json.dumps({u"Err": six.text_type(msg)})
 
+            # mutually exclusive options check
+            mutually_exclusive_list = ['virtualCopyOf', 'cloneOf', 'qos-name']
+            input_list = contents['Opts'].keys()
```

Comment:
@sonivi This will fail in python3, Satish's code is already merged that is what i can see

Reply:
@sonivi can you also modify create_help.txt to accommodate information on import. Rest changes looks fine for me

Reply:
comment addressed
Also addded unit test case